pkgpanda/build/__init__.py

```diff
@@ -958,11 +958,12 @@ def cache_abs(filename):
             pkg_path = repository.package_path(pkg_id_str)
             pkg_tar = pkg_id_str + '.tar.xz'
             if not os.path.exists(package_store.get_package_cache_folder(requires_name) + '/' + pkg_tar):
-                raise BuildError("The build tarball {} refered to by the last_build file of the "
-                                 "dependency {} variant {} doesn't exist. Rebuild the dependency.".format(
-                                    pkg_tar,
-                                    requires_name,
-                                    requires_variant))
+                raise BuildError(
+                    "The build tarball {} refered to by the last_build file of the dependency {} "
+                    "variant {} doesn't exist. Rebuild the dependency.".format(
+                        pkg_tar,
+                        requires_name,
+                        requires_variant))
```

Comment:
So both of those are accepted here. I tend to really, really hate the prior and hence moved lots of stuff to the latter. (The prior ends up with excessive wrapping for no real value)